src/animation.h

```diff
@@ -148,53 +148,66 @@ In practice the limiter value is always 1 or the animation length, meaning that
 There's never a situation where you will choose the animation timer until it hits X and then swap to 0.
 Its always one or the other, never swapping.
 
-The first triplet is used in func_80004C30 to access the Type 2 array and the values accessed is placed into D_80162D70.
-D_80162D70 is a Vec3s is then used to set the values of some Vec3f in func_80004A1C.
-All further triplets are used in func_80004A1C to collect Type 2 values to another Vec3s local to the function.
+The first triplet is used in render_armature to access the Type 2 array and the values accessed is placed into sOriginalPosAnimation.
+sOriginalPosAnimation is a Vec3s is then used to set the values of some Vec3f in render_limb_or_add_mtx.
+All further triplets are used in render_limb_or_add_mtx to collect Type 2 values to another Vec3s local to the function.
 Both the local Vec3s and Vec3f are used to create a Mat4, which is then converted to a Mtx, which is then pushed into the matrix pool.
 
-The chosen values are then used to access the Type 2 array and the value accessed is placed into D_80162D70.
-D_80162D70 is a Vec3s that is used to set values in the some Vec3f that goes on to be used to modify the same Mat4 that the `thing` values are placed into.
+The chosen values are then used to access the Type 2 array and the value accessed is placed into sOriginalPosAnimation.
+sOriginalPosAnimation is a Vec3s that is used to set values in the some Vec3f that goes on to be used to modify the same Mat4 that the `thing` values are placed into.
 
 */
 
+#define SIMPLE_ANIMATION_INSTRUCTION(x) x, 0x00000002
+
+#define ANIMATION_DISABLE_AUTOMATIC_POP SIMPLE_ANIMATION_INSTRUCTION(DISABLE_AUTOMATIC_POP_MATRIX)
+#define ANIMATION_POP_MATRIX SIMPLE_ANIMATION_INSTRUCTION(POP_MATRIX)
+#define ANIMATION_STOP SIMPLE_ANIMATION_INSTRUCTION(STOP_ANIMATION)
+//! the pos it's very often relative to the previous render
```

Comment:
pos is often relative to the previous render frame? (I think this is clearer wording)

Reply:
oh my bad on the formulation I mean more previous RENDER_MODEL_OR_ADD_POS push